pkg/lib/notifications.js

```diff
@@ -20,10 +20,10 @@
 /* NOTIFICATIONS
 
 A page can broadcast notifications to the rest of Cockpit.  For
-example, the "Software Updates" page can send out a notification when
+example, the "Software updates" page can send out a notification when
 it detects that software updates are available.  The shell will then
-highlight the menu entry for "Software Updates" and the "System"
-overview page will also mention it in its "Operating System" section.
+highlight the menu entry for "Software updates" and the "System"
```

Comment:
If we do adopt the navigation should have Title Case suggestion from PF4, then these should be kept as-is. (It's a comment, so it doesn't really matter much though, but still.)